configs/Penguin-XeonPlatinum-OmniPath/system_definition.yaml

```
@@ -0,0 +1,32 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: Penguin-XeonPlatinum-OmniPath
+  manufacturer:
+    vendor: PenguinComputing
+    name: RelionCluster
+  processor:
+    vendor: Intel
+    name: XeonPlatinum924248C
+    ISA: x86_64
+    uArch: IceLake-SP
```

```suggestion
    uArch: icelake
```
to match to `archspec` monikers instead of whatever marketing things they come up with.